release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -328,6 +328,11 @@ For the OVN-Kubernetes network plug-in, egress firewalls support audit logging u
 [id="ocp-4-12-machine-api"]
 === Machine API
 
+[id="ocp-4-12-mapi-control-plane-machine-sets"]
+==== Control plane machine sets 
+
+{product-title} 4.12 introduces control plane machine sets. Control plane machine sets provide for control plane machines similar management capabilities as compute machine sets provide for compute machines. For more information, see the xref:../machine_management/control_plane_machine_management/cpmso-about.adoc#cpmso-about[Managing control plane machines].
```

Comment:
Just a rewording suggestion that I think makes this a little easier to parse:
```suggestion
{product-title} 4.12 introduces control plane machine sets. Control plane machine sets provide management capabilities for control plane machines that are similar to what compute machine sets provide for compute machines. For more information, see the xref:../machine_management/control_plane_machine_management/cpmso-about.adoc#cpmso-about[Managing control plane machines].
```

Reply:
ty Jesse, I like it :)